app/_includes/snippets/gateway-operator/install_with_helm.md

```diff
@@ -10,12 +10,32 @@ helm repo add kong https://charts.konghq.com
 helm repo update kong
 ```
 
-Install {{ site.kgo_product_name }} with Helm:
+Install {{ site.kgo_product_name == "true"}} with Helm:
```

Comment:
`kgo_product_name == "true"` 🤔 ?

Reply:
```suggestion
Install {{ site.kgo_product_name }} with Helm:
```
I'm going to assume copy/paste error, since that doesn't do anything.

Reply:
Yep, thanks both for catching it :)